device/arista/x86_64-arista_7050_qx32/Arista-7050-QX32/bin/eeprom.py

```diff
@@ -0,0 +1,220 @@
+#!/usr/bin/env python
```

Comment:
eeprom.py and sfputil.py should be under onie_platform directory as it is hardware platform specific and does not depends on the port breakout mode.